charts/promitor-agent-scraper/templates/clusterrolebinding.yaml

```diff
@@ -14,6 +14,10 @@ roleRef:
   name: {{ template "promitor-agent-scraper.fullname" . }}
 subjects:
   - kind: ServiceAccount
-    name: {{ template "promitor-agent-scraper.serviceAccountName" . }}
+    {{- if .Values.rbac.serviceAccount.create }}
```

Comment:
Doesn't this logic have to be reversed?

If `.Values.rbac.serviceAccount.create` is true, then use `.Values.rbac.serviceAccount.name`

Reply:
Actually, we have to check if `.Values.rbac.serviceAccount.name` is specified and use that, if not use `{{ template "promitor-agent-scraper.fullname" . }}`